lib/findingaids/ead/behaviors.rb

```diff
@@ -114,6 +125,10 @@ def ead_language_fields fields = Hash.new
     return fields
   end
 
+  def fix_subfield_demarcators(value)
+    v = value.gsub(/\|\w{1}/,"--")
+    return v
+  end
```

Comment:
@eshadatta This can be simplified to a one liner, and please add comments like the following:

```ruby 
##
# Replace MARC style subfield demarcators
#
# Usage: fix_subfield_demarcators("Subject 1 |z Sub-Subject 2") => "Subject 1 -- Sub-Subject 2"
def fix_subfield_demarcators(value)
   value.gsub(/\|\w/,'--')
end
```